### ISSUE TITLE
Feature request: tutorial videos for creating nodes

### ISSUE THREAD

**Issue by trondarild** [leon-thomm/Ryven#32]
Although there is instructions for creating nodes at the website, it would be useful to have short videos showing how to get started programming simple nodes, and importing them into Ryven.

Thanks!

**Comment by leon-thomm** [comment]
I totally agree. I am working on a huge extension of the project which seems to have a lot of potential so far, but which would be released in a few months at earliest, in the end probably leading to a release of Ryven 3. That's why I want to wait a bit longer, because I can't really tell what changes will follow during the next weeks. But tutorials are planned.

**Comment by trondarild** [comment]
Great! Looking forward to it!

**Comment by charoduke** [comment]
I can create simple nodes.
But I want to understand how to make a node with some default input values.
I can use obj in console like `obj.inputs[0].widget.setText('bla')` e.t.c.
While I'm try it in "__init__" method in node get error because  self.inputs empty.
